playbooks/roles/consul/tasks/acl.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: Install Consul ACL apt requirements
+  apt:
+    name:
+      - python3-pip
+
+- name: Install Consul ACL python requirements
+  pip:
+    executable: pip3
+    name:
+      - python-consul==1.1.0
+      - pyhcl==0.3.10
+
+- name: Configure Consul ACL anonymous token
+  consul_acl:
+    token: anonymous
+    mgmt_token: "{{ consul_acl_master_token }}"
+    rules: "{{ consul_acl_anonymous_policy }}"
+  when: consul_server and consul_acl_anonymous_policy is not none
```

Comment:
I actually already did this step manually because it only needs to be done one time per cluster, but wanted to automate it for the future in Ansible.

To minimize what hosts it runs on, I put `consul_server` as one of the requirements. I also tried `run_once: true` but then if we deploy to all HAProxy servers (which includes haproxy.opencraft.hosting, which is not a `consul_server`), there's a chance it wouldn't get run at all. To not complicate it I just left it like this instead - it will run on all Consul servers, and do nothing if it's already there, yep.

Reply:
Thanks for the explanation – sounds all good to me.